lib/cocoapods-packager/builder.rb

```diff
@@ -110,9 +110,16 @@ def copy_headers
       headers_source_root = "#{@public_headers_root}/#{@spec.name}"
 
       Dir.glob("#{headers_source_root}/**/*.h").
-        each { |h| `ditto #{h} #{@fwk.headers_path}/#{h.sub(headers_source_root, '')}` }
-
-      if File.exist?("#{@public_headers_root}/#{@spec.name}/#{@spec.name}.h")
+          each { |h| `ditto #{h} #{@fwk.headers_path}/#{h.sub(headers_source_root, '')}` }
+
+      # if custom 'module_map' is specified add it
+      # to the framework distribution, otherwise check
+      # if a header exists that is equal to 'spec.name'', if so
+      # create a default 'module-map' one using it.
+      if !@spec.module_map.nil?
+        module_map_file = "#{@sandbox_root}/#{@spec.name}/#{@spec.module_map}"
```

Comment:
thanks @segiddins for commenting!

I see the [module_map method](http://www.rubydoc.info/gems/cocoapods/Pod/Sandbox/FileAccessor#module_map-instance_method) can be used  but I am unable to find a proper way to inject access to a ```FileAccessor``` object in-this class context. Mind to give me a hint on it?